modules/security/keyvault/version.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
+  "version": "0.0",
```

Comment:
Do we want to start with version `0.0.1`?

Reply:
```suggestion
  "version": "0.0.1",
```